getting-started/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting started with Quarkus
+
+This is a minimal CRUD service exposing a couple of endpoints over REST.
+
+Under the hood, this demo uses:
+
+- RESTEasy to expose the REST endpoints
+- REST-assured and JUnit 5 for endpoint testing
+
+## Requirements
+
+To compile and run this demo you will need:
+
+- JDK 1.8+
+- GraalVM
+- Apache Maven `3.5.3+`
```

Comment:
I would drop this and ask the user to use the Maven wrapper included in the quickstart. One less requirement to change in the future :).